src/Tizen.NUI.Components/Controls/ItemsView/CollectionView.cs

```diff
@@ -0,0 +1,815 @@
+        /// old selection list and new selection will be provided.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+		public event EventHandler<SelectionChangedEventArgs> SelectionChanged;
+
+		bool _suppressSelectionChangeNotification;
+
+        /// <summary>
+        /// Size strategy of measuring scroll content. see details in ItemSizingStrategy.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public ItemSizingStrategy SizingStrategy;
+        private List<PropertyNotification> notifications = new List<PropertyNotification>();
+        
+        /// <summary>
+        /// Base constructor.
+        /// </summary>
+        /// <since_tizen> 6.5 </since_tizen>
+        /// This may be public opened in tizen_6.0 after ACR done. Before ACR, need to be hidden as inhouse API
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public CollectionView() : base()
+        {
+            FocusGroup = true;
+            Init();
+            SetKeyboardNavigationSupport(true);
+            Scrolling += OnScrolling;
+        }
+
+        private void Init()
+        {
+            if (ItemsSource == null) return;
+            if (ItemsLayouter == null) return;
+            if (ItemsTemplate == null) return;
+
+            InternalItemSource.HasHeader = (header != null);
+            InternalItemSource.HasFooter = (footer != null);
+
+            ItemsLayouter.Initialize(this);
+            ItemsLayouter.RequestLayout(0.0f, true);
+
+            if (ScrollingDirection == Direction.Horizontal)
+            {
+                ContentContainer.SizeWidth = ItemsLayouter.CalculateLayoutOrientationSize();
+            }
+            else
+            {
+                ContentContainer.SizeHeight = ItemsLayouter.CalculateLayoutOrientationSize();
+            }
+        }
+
+        // Realize and Decorate the item.
+        internal override ViewItem RealizeItem(int Index)
+        {
+            if (Index == 0 && Header != null)
+            {
+                Console.WriteLine("Header is Showing!");
+                Header.Show();
+                return Header;
+            }
+
+            if (Index == InternalItemSource.Count - 1 && Footer != null)
+            {
+                Footer.Show();
+                return Footer;
+            }
+
+            ViewItem item = base.RealizeItem(Index);
+
+            item.Template = (ItemsTemplate is DataTemplateSelector ?
+                            (ItemsTemplate as DataTemplateSelector).SelectDataTemplate(InternalItemSource.GetItem(Index), this) :
+                             ItemsTemplate);
```

Comment:
`?.`와 `??`를 이용하여 간단히 줄일 수 있습니다.
```suggestion
            item.Template = (ItemsTemplate as DataTemplateSelector)?.SelectDataTemplate(InternalItemSource.GetItem(Index), this) ?? ItemsTemplate;
```